src/EditorFeatures/CSharpTest/Interactive/NavigateTo/InteractiveNavigateToTests.cs

```diff
@@ -268,7 +268,7 @@ class Foo
             {
                 SetupVerifiableGlyph(StandardGlyphGroup.GlyphGroupProperty, StandardGlyphItem.GlyphItemPublic);
                 var item = (await _aggregator.GetItemsAsync("this")).Single();
-                VerifyNavigateToResultItem(item, "this[]", MatchKind.Exact, NavigateToItemKind.Property, displayName: "this[int]", additionalInfo: $"{FeaturesResources.type_space}Foo");
+                VerifyNavigateToResultItem(item, "this", MatchKind.Exact, NavigateToItemKind.Property, displayName: "this[int]", additionalInfo: $"{FeaturesResources.type_space}Foo");
```

Comment:
The first string ("this") is what we store internally for the "name" of a indexer in our indexer.  It must always be a prefix of the final "displayName" we show.  Otherwise, any matched highlight spans in it won't be correct in the display name.